service/controller/v14/resource/cloudformation/error.go

```diff
@@ -56,6 +56,27 @@ func IsNotFound(err error) bool {
 	return microerror.Cause(err) == notFoundError
 }
 
+var resourceNotReadyError = microerror.New("resource not ready")
+
+// IsResourceNotReady asserts resourceNotReadyError.
+func IsResourceNotReady(err error) bool {
+	c := microerror.Cause(err)
+
+	if c == nil {
+		return false
+	}
+
+	if strings.Contains(c.Error(), "ResourceNotReady") {
```

Comment:
I checked the godoc for the CF SDK but there is nothing we can use so I have to string match here. :(